modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v1_1_0/UpgradeDDMStructure.java

```diff
@@ -159,6 +159,8 @@ private String _convertExpression(String visibilityExpression) {
 			ExpressionParameterValueExtractor.extractParameterValues(
 				visibilityExpression);
 
+		StringBundler sb1 = new StringBundler();
```

Comment:
@rodrigopaulino, if I've understood correctly, this commit is adding support to multiple expressions in a Visibility Expression, like `contains(text, "show") || contains(text, "appear")`, am I right?

Reply:
Mainly yes, @natocesarrego. But not all compound expressions will work eventually (for example, different logical operators in the same expression). For those other cases I've created [this LPS](https://issues.liferay.com/browse/LPS-125200), which requires an intricate solution to translate the visibility expression grammar as users knew back in 7.0 to the current grammar used in 7.1+.